addons/gdUnit3/src/update/GdUnitUpdate.gd

```diff
@@ -102,7 +78,7 @@ func extract_releases(response :HttpResponse, current_version :GdUnit3Version) -
 		if GdUnit3Version.parse(release["tag_name"]).equals(current_version):
 			break
 		var release_description :String = release["body"]
-		var bbcode = convert_markdown2bbcode(release_description)
+		var bbcode = _md_reader.tobbcode(release_description)
```

Comment:
Change `to_bbcode` ?

Reply:
yes it is changed to `to_bbcode` 
The branch is work in progress. i pushed my last changes only.